edbterraform/data/terraform/aws/modules/kubernetes/outputs.tf

```diff
@@ -0,0 +1,30 @@
+output "cluster_id" {
+  description = "EKS cluster ID"
+  value       = module.eks.cluster_id
+}
+
+output "cluster_endpoint" {
+  description = "Endpoint for EKS control plane"
+  value       = module.eks.cluster_endpoint
+}
+
+output "cluster_security_group_id" {
+  description = "Security group ids attached to the cluster control plane"
+  value       = module.eks.cluster_security_group_id
+}
+
+output "cluster_certificate" {
```

Comment:
This output is named `ca_certification` under google's gke. This should be updated to either match googles outputs or googles outputs should be changed to match eks.

https://github.com/EnterpriseDB/edb-terraform/blob/e9f4504d27f4c7ddd688422022d845d7ce5bea0b/edbterraform/data/terraform/azure/modules/kubernetes/outputs.tf#L11

Reply:
Updated to ca_certification

Reply:
@dougortiz if you could update this to `ca_certificate` to match azure's aks outputs. 

I referenced `ca_certification` and google's gke in the last review but that was a typo on my part and mentioned the wrong provider. Link below shows what azure's aks output has.

https://github.com/EnterpriseDB/edb-terraform/blob/e9f4504d27f4c7ddd688422022d845d7ce5bea0b/edbterraform/data/terraform/azure/modules/kubernetes/outputs.tf#L11

Reply:
Renamed and updated to: 'ca_certificate' in outputs.tf